qa/automation/CareerPortal.Tests.Scenarios/Savostin/buttons.feature

```diff
@@ -0,0 +1,52 @@
+Feature: 1.0.0 buttons lead to correct pages
```

Comment:
As I have noticed, you decided to check the main page on Quantori website. I would recommend to specify it in feature name, so that we know what is exactly going on here. :)